package.json

```diff
@@ -55,8 +56,8 @@
     "@astrojs/webapi": "workspace:*"
   },
   "devDependencies": {
-    "@changesets/changelog-github": "^0.4.4",
-    "@changesets/cli": "^2.22.0",
+    "@changesets/changelog-github": "0.4.4",
```

Comment:
hardcode to this version to be safe, so that nothing changes in the implementation over the next 2 months.